practices/commit_naming_rules.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: commit-naming-rules
+stages:
+  - development
+short_description: Having a commit naming convention can be of great help, in maintaining the codebase and easing up the development process. Find out more about the good practices around commit naming rules!
+tags:
+  - commit messages
+  - commit names
+  - commits
+  - git
+  - github
+  - development process
+keywords:
+  - commits
+  - commit messages
+  - commit names
+  - git
+  - github
+  - development process
+  - commit rules
+  - commit best practices
+covered_by_dx_scanner: true
+---
+
+# Commit Naming Rules
+
+**TL;DR**
+
+Having a commit naming convention can be of great help, in maintaining the codebase and easing up the development process. Find out more about the good practices around commit naming rules!
+
+![Commit naming guide](/files/commit_guide.png)
+
```

Comment:
Please, add which problems Commit Naming Rules (as practice) solves. If you could, include some of the already covered issues on [DX Knowledgebase](https://developerexperience.io/problems)